rest/config_test.go

```diff
@@ -2529,3 +2529,85 @@ func TestBucketCredentialsValidation(t *testing.T) {
 		})
 	}
 }
+
+// Tests behaviour of CBG-2257 to poll only buckets in BucketCredentials that don't currently have a database
+func TestServerlessPollBuckets(t *testing.T) {
```

Comment:
Should the 'poll only buckets that don't have a database' part of this wait until CBG-2280 is completed?  Or at least be based on a flag that's enabled when CBG-2280 is done?  Otherwise running in serverless mode isn't going to pick up changes to database configurations.

Reply:
Commented out the code that does this so it can be enabled as part of CBG-2280